taxi/views.py

```diff
@@ -85,6 +111,20 @@
     model = Driver
     paginate_by = 5
```

Comment:
The `paginate_by` attribute is correctly set to enable pagination. Ensure that the template handles pagination controls properly.